dependencies/pom.xml

```diff
@@ -962,6 +966,53 @@
                 <artifactId>kafka-clients</artifactId>
                 <version>${version.lib.kafka}</version>
             </dependency>
+            <!-- Langchain4j integration -->
+            <dependency>
+                <groupId>dev.langchain4j</groupId>
+                <artifactId>langchain4j-cohere</artifactId>
```

Comment:
Can we use `langchain4j-bom` to manage versions?

Reply:
Different providers use different dependencies. We don't want to include all dependencies because it doesn't make sense. For example, if you are using Open AI it doesn't make sense to have Ollama in the classpath.

Reply:
Created #9746 to collect follow ups

Reply:
@m0mus Boms manage versions -- they don't add dependencies.